src/utils/utils.py

```diff
@@ -168,8 +168,7 @@ def log_hyperparameters(object_dict: dict) -> None:
     hparams["ckpt_path"] = cfg.get("ckpt_path")
     hparams["seed"] = cfg.get("seed")
 
-    # send hparams to all loggers
-    trainer.logger.log_hyperparams(hparams)
+    return hparams
```

Comment:
I think we can still send hyperparameters here instead of returning:
```python
for logger in trainer.loggers:
    logger.log_hyperparams(hparams)
```

Reply:
Yes, I agree, this way there is less code to modify.